### PR TITLE
mickeypash's Bulgarian Phrases Cheat Sheet Goodie

### PR DESCRIPTION
**Pull request by mickeypash**
# Bulgarian cheat sheet

**What does your Instant Answer do?**
Provides a quick reference of beginner phrases for travelers in Bulgaria.

**What is the data source for your Instant Answer? (Provide a link if possible)**

[Linguant](http://www.linguanaut.com/english_bulgarian.htm)
I am a native speaker.

**Why did you choose this data source?**

It seemed straightforward and I based it on the current German Cheat Sheet so as to meet the standard.

**Are there any other alternative (better) data sources?**

There are certain more comprehensive sources but they require knowledge of linguistics.

[Omniglot](http://www.omniglot.com/language/phrases/bulgarian.php)
[Wikinationary](https://en.wiktionary.org/wiki/Appendix:Basic_Bulgarian_glossary)
[Wikitravel](http://wikitravel.org/en/Bulgarian_phrasebook)
[BBC languages - MP3](http://www.bbc.co.uk/languages/other/quickfix/bulgarian.shtml) 

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**

It will primarily be useful for people travelling to Bulgaria.
Additionally to individuals wanting to surprise/impress their Bulgarian friends.

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**

Not as far as I know.

**Are you having any problems? Do you need our help with anything?**

Not at them moment.

**Where did you hear about DuckDuckHack? (For first time contributors)**

I was researching the website as I am interested in Information Retrieval as well as contributing to Open-Source initiatives.

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
[Screenshot](http://i.imgur.com/oqAB5Jh.png?1)

# Checklist

```
- [x] Added metadata and attribution information (https://duck.co/duckduckhack/metadata)
- [] Wrote test file and added to t/ directory (https://duck.co/duckduckhack/goodie_tests)
- [x] Verified that instant answer adheres to design guidelines (https://duck.co/duckduckhack/design_styleguide)
- [x] Verified that instant answer adheres to code styleguide (https://duck.co/duckduckhack/code_styleguide)
- [x] Tested cross-browser compatibility

    Please let us know which browsers/devices you've tested on:
    - Windows 8
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 10

    - Windows 7
        - [x] Google Chrome
        - [] Firefox
        - [x] Opera
        - [] IE 8
        - [] IE 9
        - [] IE 10

    - Windows XP
        - [] IE 7
        - [] IE 8

    - Mac OSX
        - [x] Google Chrome
        - [] Firefox
        - [] Opera
        - [x] Safari

    - iOS (iPhone)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - iOS (iPad)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - Android
        - [] Firefox
        - [] Native Browser
        - [] Google Chrome
        - [] Opera
```

-----
IA Page: https://duck.co/ia/view/bulgarian_cheat_sheet

### CODE REVIEW

**Comment by moollaza** [comment]
@mickeypash thanks a lot for contributing, we really appreciate it! We'll be sure to review this shortly and give you some feedback!

I'll have one our community translation mangers verify the language and then we can go from there!

**Comment by andrey-p** [comment]
Not a community translation manager, but here's a few things I noticed:

- [`dobur wecher`](https://github.com/mickeypash/zeroclickinfo-goodies/blob/master/share/goodie/cheat_sheets/bulgarian.json#L27) should be `dobur vecher`
- incorrect spacing on [`dobredo shli`](https://github.com/mickeypash/zeroclickinfo-goodies/blob/master/share/goodie/cheat_sheets/bulgarian.json#L131) - should be `dobre doshli`
- [`vidq menu`](https://github.com/mickeypash/zeroclickinfo-goodies/blob/master/share/goodie/cheat_sheets/bulgarian.json#L131) - should be `vidia menu`

And some things based on personal opinion, feel free to disagree :)

- `ъ` might be better transcribed as `a` instead of `u` to make easier to pronounce phonetically
- `я` might be better as `ya` rather than `ia` to make it clear that it needs to be pronounced like the `ya` in `yak`, as opposed to the `ia` in `pariah`.

Looking pretty good otherwise!

**Comment by mickeypash** [comment]
@andrey-p Good spot! Thank you :+1:

**Comment by moollaza** [comment]
@mickeypash apologies for the delay!

I'll be putting the language cheat sheets onto beta.duckduckgo.com very soon and we'll then solicit feedback from the community on whether or not they approve of the translations. Once that's done and we've gathered some feedback, we'll need to implement any necessary changes and then this will be ready for merge.

File: lib/DDG/Goodie/IsAwesome/mickeypash.pm [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package DDG::Goodie::IsAwesome::mickeypash;
```
Comment (moollaza) [inline lib/DDG/Goodie/IsAwesome/mickeypash.pm]: Whoops, this shouldn't be here. Please remove this file from the PR.

File: t/IsAwesome/mickeypash.t [hunk 2]
```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env perl
```
Comment (moollaza) [inline t/IsAwesome/mickeypash.t]: Whoops, this shouldn't be here. Please remove this file from the PR.

File: share/goodie/cheat_sheets/bulgarian.json [hunk 3]
```diff
@@ -0,0 +1,232 @@
+{
+   "id" : "bulgarian_cheat_sheet",
+   "name" : "Bulgarian Cheat Sheet",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/bulgarian.json]: Change to `Bulgarian Phrases`

**Comment by moollaza** [comment]
@mickeypash Ping! Sorry this has taken so long!

I'm just checking in to see if you're still interested in finishing this one up? We'd love to see this go live!

There's absolutely no rush, just wanted to make sure you're not stuck on anything.

**Comment by mickeypash** [comment]
Hey @moollaza Still here! Will get ti done now :)

**Comment by moollaza** [comment]
@mickeypash congrats! It's live: https://duckduckgo.com/?q=Bulgarian+cheat+sheet&ia=cheatsheet

It's actually been live for a few weeks now!

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Pull request merged**